test/Dapper.AOT.Test/Interceptors/QueryCustomConstruction.input.cs

```diff
@@ -0,0 +1,127 @@
+﻿using Dapper;
+using System.Data.Common;
+
+[module: DapperAot]
+
+public static class Foo
```

Comment:
propose scenario:

- get-only properties via contructor
- record class
- record struct

Reply:
supported get-only properties via ctor

Reply:
and supported short syntax of ctors for `record`s and `record struct`s:
```
public record RecordClassSimpleCtor(int X, string Y, double? Z);
public record struct RecordStructSimpleCtor(int X, string Y, double? Z);
```